go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/replicatedhq/kurlkinds v1.0.10
 	github.com/replicatedhq/pvmigrate v0.7.0
 	github.com/replicatedhq/troubleshoot v0.53.0
+	github.com/ricardomaraschini/plumber v1.14.0
```

Comment:
Shouldn't this be github.com/replicatedhq/plumber v1.15.0 ?

Reply:
I will change it but i will need to replace in the go mod because module declares itself as github.com/ricardomaraschini/plumber

Reply:
I have renamed the project in our repository, should be good to go in v1.16.0. It turns out that the idea of just forking was not a good one, I am sorry.  I think that the best course of action here would be to drop the replicatedhq repository and then I can transfer ownership of the original repo. What do you think ?